multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -260,6 +265,8 @@ def _fit_x(self, X: pd.DataFrame):
                 self._ignore_columns_set.add(col_name)
             elif col_type.startswith(TEXT):
                 self._text_feature_names.append(col_name)
+            elif col_type.startswith(NER):
+                self._ner_feature_names.append(col_name)
```

Comment:
`_ner_feature_names ` seems always empty after calling `_fix_x()`.

Reply:
it's not empty as we require the users to set column_types for multimodal ner.